### ISSUE TITLE
Cannot launch new workflow in v.0.18.0 helm chart

### ISSUE THREAD

**Issue by AndriySidliarskiy** [argoproj/argo-helm#1468]
### Describe the bug

Submit new workflow but nothing doing. In controller logs get request is doing but controller have Liveness probe failed: HTTP probe failed with statuscode: 500 and workflow not launching. Can you help or solve this please .

### Related helm chart

argo-workflows

### Helm chart version

0.18.0

### To Reproduce

1. Use argo ui 
2. Run new worklfow

### Expected behavior

Run new workflow

### Screenshots

<img width="1390" alt="image" src="https://user-images.githubusercontent.com/62854979/191201482-6f070bb8-9385-41ef-8618-2e87040d27cb.png">

### Additional context

UPD: Solve this problem when redeploy helm chart but update to new version not working

**Comment by jmeridth** [comment]
@AndriySidliarskiy Do you have any logs from the controller that can give us a better idea of what is going on?  I'm retesting the helm chart locally right now.

**Comment by chr-b** [comment]
I am having the same issue.

The workflow controller logs shows:
```
W0921 08:44:42.110191       1 reflector.go:324] pkg/mod/k8s.io/client-go@v0.24.3/tools/cache/reflector.go:167: failed to list *v1alpha1.WorkflowArtifactGCTask: the server could not find the requested resource (get workflowartifactgctasks.argoproj.io)
E0921 08:44:42.110226       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.24.3/tools/cache/reflector.go:167: Failed to watch *v1alpha1.WorkflowArtifactGCTask: failed to list *v1alpha1.WorkflowArtifactGCTask: the server could not find the requested resource (get workflowartifactgctasks.argoproj.io)
```

This is probably related to #1430:
```
$ kubectl get crd
...
workfloweventbindings.argoproj.io                2021-12-20T11:05:31Z
workflows.argoproj.io                            2021-12-20T11:05:32Z
workflowtasksets.argoproj.io                     2021-12-20T11:05:32Z
workflowtemplates.argoproj.io                    2021-12-20T11:05:33Z
```
Those might be left over from a previous install, which means that the new CRDs are not installed.

Using Terraform provider `2.6.0` that is based on Helm `3.9.0`.

**Comment by jmeridth** [comment]
@chr-b yeah, the issue seems to be doing this through the helm terraform provider.  My local testing has all CRDs installing on fresh cluster. That's not using terraform helm provider. I'll test with that. Thank you for filing issue and possibly noticing link to #1430. Cheers.

**Comment by jmeridth** [comment]
@chr-b after doing some research, looks like the terraform helm provider and helm have an ongoing issue re: the crds folder.  I've created a PR to move the crds folder into the templates folder and match the technique of the other helm charts we manage re: conditionals to install and keep CRDs. /cc @christophercutajar

Thank you both for your patience and filing these issues.  Once this PR gets merged I'd appreciate a test of your terraform usage to confirm this fixes these issues.  If it does not, please re-open them.

**Comment by jmeridth** [comment]
#1472 merged.  Closing this.  If you get latest chart 0.19.0 and still experience this issue, please let me know.  Cheers.

**Comment by chr-b** [comment]
Thanks @jmeridth, the issue is fixed with chart `0.19.0`.

**Comment by christophercutajar** [comment]
@jmeridth thanks for fixing this! Was caught up in something else and hadn't time to test with Terraform! @chr-b thanks for confirming - I'll spend sometime during the weekend testing this too!
